scripts/train.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Trains a model with AutoML Tables using parameters specified in the
+config. Training typically completes in one hour more than that specified
+in the train budget, though compute is only charged for the budgeted time.
+See https://cloud.google.com/automl-tables/docs/train for details.
+"""
+
+import logging
+import sys
+
+from google.cloud import automl_v1beta1 as automl
+
+import utils
+
```

Comment:
Double empty lines before and after logging, two blank lines for top level declarations.